packages/auto-label/test/auto-label.ts

```diff
@@ -567,6 +567,144 @@ describe('auto-label', () => {
       });
       ghRequests.done();
     });
+
+    it('does not updates staleness labels when same', async () => {
```

Comment:
Tests are looking great, I think it would be worth adding one additional assertion that, kif `staleness` is not set in config, labels are not added to a PR that would otherwise have them added with the default settings.

Reply:
thanks - added tests